web/settings.py

```
@@ -3,20 +3,21 @@
 from pathlib import Path
 
 import environ
+from cryptography.fernet import Fernet
 
 BASE_DIR = Path(__file__).resolve().parent.parent
 
 SECRET_KEY = "django-insecure-5kyff0s@l_##j3jawec5@b%!^^e(j7v)ouj4b7q6kru#o#a)o3"
```

Ruff (0.8.2)

Possible hardcoded password assigned to: "SECRET_KEY"

(S105)